seedlet/functions.php

```diff
@@ -353,8 +311,8 @@ function seedlet_content_width() {
  * Enqueue scripts and styles.
  */
 function seedlet_scripts() {
-	// Enqueue Google fonts
-	wp_enqueue_style( 'seedlet-fonts', seedlet_fonts_url(), array(), null );
+	// Fonts 
+	wp_enqueue_style( 'seedlet-fonts', get_template_directory_uri() . '/assets/css/fonts.css', array(), wp_get_theme()->get( 'Version' ) );
```

Comment:
This is a little trickier than I thought. Didn’t realize the URI’s were different for the editor but I think its easier to override a singular font url function. Otherwise, I worry we’ll be forcing child themes to bundle fonts which isn’t really something we can do with the build tool, and can be cumbersome to do repetitively in child themes when we may want simply load external Google fonts. 

Can we do this like we do in Varia themes or is that bad for the TRT? 
https://github.com/Automattic/themes/blob/36c24e223f03a166e0548a2c3ed6bb0ee9464cce/alves/functions.php#L218-L233

That way we can override a singular function making it easier for us to override with a Google font _or_ a bundled font.

Reply:
I don't see why we can't do it the way you're suggesting, I will try it out.

Reply:
@allancole check out https://github.com/Automattic/themes/pull/2251/commits/fa4cb62801afb293de1e0048101c9e1cab2a47d9 — is this what you had in mind?